articles/partner-twilio-php-make-phone-call.md

```diff
@@ -24,76 +24,80 @@ The following example shows you how you can use Twilio to make a call from a PHP
-         <tr>
-               <td colspan=2><input type="submit" value="Make this call"></td>
-         </tr>
-       </table>
-     </form>
-     <br/>
-    </body>
-    </html>
+```html
+<html>
+<head>
+    <title>Automated call form</title>
+</head>
+<body>
+<h1>Automated Call Form</h1>
+ <p>Fill in all fields and click <b>Make this call</b>.</p>
+  <form action="makecall.php" method="post">
+   <table>
+     <tr>
+           <td>To:</td>
+           <td><input type="text" size=50 name="callTo" value=""></td>
+     </tr>
+     <tr>
+           <td>From:</td>
+           <td><input type="text" size=50 name="callFrom" value=""></td>
+     </tr>
+     <tr>
+           <td>Call message:</td>
+           <td><input type="text" size=100 name="callText" value="Hello. This is the call text. Good bye." /></td>
+     </tr>
+     <tr>
+           <td colspan=2><input type="submit" value="Make this call"></td>
+     </tr>
+   </table>
+ </form>
+ <br/>
+</body>
+</html>
+```
 
 ## Create the code to make the call
-The following code shows how to build a web page (**makecall.php**) which is called when the user submits the form displayed by **callform.html**. The code shown below creates the call message and generates the call. (Use your Twilio account and authentication token instead of the placeholder values assigned to **$sid** and **$token** in the code below.)
+The following code shows how to build **makecall.php**, which is called when the user submits the form displayed by **callform.html**. The code shown below creates the call message and generates the call. Also, be sure to use your Twilio account and authentication token from the [Twilio Console][twilio_console] instead of the placeholder values assigned to **$sid** and **$token** in the code below.
 
-    <html>
-    <head><title>Making call...</title></head>
-    <body>
-    <p>Your call is being made.</p>
+```php
+<html>
+<head><title>Making call...</title></head>
+<body>
+<p>Your call is being made.</p>
 
-    <?php
-    require_once 'Services/Twilio.php';
+<?php
+require_once 'Services/Twilio.php';
 
-    $sid = "your_account_sid";
-    $token = "your_authentication_token";
+$sid = "your_account_sid";
+$token = "your_authentication_token";
 
-    $from_number = $_POST['callFrom']; // Calls must be made from a registered Twilio number.
-    $to_number = $_POST['callTo'];
-    $message = $_POST['callText'];
+$from_number = $_POST['callFrom']; // Calls must be made from a registered Twilio number.
+$to_number = $_POST['callTo'];
+$message = $_POST['callText'];
 
-    $client = new Services_Twilio($sid, $token, "2010-04-01");
+$client = new Services_Twilio($sid, $token, "2010-04-01");
 
-    $call = $client->account->calls->create(
-        $from_number,
-        $to_number,
-          'http://twimlets.com/message?Message='.urlencode($message)
-    );
+$call = $client->account->calls->create(
+    $from_number,
+    $to_number,
+      'http://twimlets.com/message?Message='.urlencode($message)
```

Comment:
I believe in V4 of _Twilio PHP Helper Library_ we are still creating calls like this: https://github.com/TwilioDevEd/api-snippets/blob/master/rest/making-calls/example-1/example-1.4.x.php#L10